astroplan/plots/mplstyles.py

```diff
@@ -0,0 +1,92 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from astropy.visualization import astropy_mpl_style
+
+__all__ = ['light_style_sheet', 'dark_style_sheet',
+           'available_style_sheets']
+
+available_styles = ["light_style_sheet", "dark_style_sheet"]
+
+def available_style_sheets():
+    """
+    Matplotlib style sheets available within astroplan.
+    """
+    return available_styles
+
+# One update specifically for astroplan.plots: raise bottom up from default 0.1
+astropy_mpl_style.update({'figure.subplot.bottom': 0.15})
```

Comment:
I'm not sure, but it looks to me like `astropy_mpl_style` is a module-level variable in `astropy.visualization`.
So if you change it here, importing this module will have the side effect to change the style for any plots that use `astropy_mpl_style` which will be very confusing for users if it happens.

How about starting by making `light_style_sheet` a copy of `astropy_mpl_style` and then modifying that copy.
And a nitpick comment: even if there's only one update at the moment, put it on a separate line so that it can be easily commented out and in (like you did for the dark_style_sheet below).